### PR TITLE
HPCC-21797 Add missing PySpark to Spark Row & Schema conversion function

### PR DESCRIPTION
**Pull request by jpmcmu**
- Modified inferSchema to maintain field order

Signed-off-by: James McMullan <James.McMullan@lexisnexis.com>

### CODE REVIEW

**Comment by jpmcmu** [comment]
@rpastrana please review

File: DataAccess/src/main/java/org/hpccsystems/spark/PySparkField.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class PySparkField 
+{
+    public PySparkField(String n, Object v)
+    {
+        name = n;
+        value = v;
+    }
+
+    public String name = null;
```
Comment (rpastrana) [inline DataAccess/src/main/java/org/hpccsystems/spark/PySparkField.java]: once constructed, should the name remain constant? Perhaps both fields should be private w/ a getter for name and get/set for val?

Comment (jpmcmu) [reply DataAccess/src/main/java/org/hpccsystems/spark/PySparkField.java]: Getters added

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**